docs/getting-started.md

```diff
@@ -67,19 +67,25 @@ directory (default: `target/${toolchain}/${mode}`, where mode can be `debug` or
 To build a kernel for the reference VMM to boot, check out the scripts in
 [resources/kernel](../resources/kernel).
 
-- [`make_busybox.sh`](../resources/kernel/make_busybox.sh) builds an ELF image
-  with a baked-in initramfs running [Busybox](https://busybox.net/). It uses a
-  stripped-down
+- [`make_kernel_image.sh`](../resources/kernel/make_kernel_image.sh) builds an
```

Comment:
Is this script the one that's ultimately renamed to `make_kernel_busybox_image` in #42? Also I guess something like the original `make_busybox` is the shortest and clearest name (especially with the additional documentation available), but if you prefer another name that's fine as well.

Reply:
Yup, it's the same script. There's already a `make_busybox` script that builds just the busybox image. This script calls into it as part of the larger kernel+initramfs build. I'm (obviously) not good with names - do you think the helper scripts should be renamed as well, so that the "user-facing" scripts keep concise names?

Reply:
Would be nice for the scripts that are user-facing to have the nicer names if you already have an approach here mind. We can have a second look at the whole thing when rebasing #42 on top of this (if that's how we are going to proceed). Meanwhile, just wondering, I see Laura's comment above about removing the `.sh` extension, but then it seems it's still present when looking at the actual commit :-s Am I missing/looking at something else here? :-s

Reply:
It wasn't a hard requirement or something like that for this particular PR. I mostly wanted to not have both `make_kernel_image.sh` and `make_kernel_image` in docs, scripts etc. If you are ok with renaming it, we can do that either in this PR or #42.